### ISSUE TITLE
Plugin enabled but no "options" page

### ISSUE THREAD

**Issue by bolololololo** [FlorianWoelki/obsidian-icon-folder#169]
I've installed the plugin but the "options" view isn't available so I can't configure it.

![image](https://user-images.githubusercontent.com/80776140/235579053-fcc6a408-737e-4388-9e8d-9c889ff565cb.png)

**Comment by FlorianWoelki** [comment]
You should see the obsidian icon folder settings when you go to settings. This is the same thing.

**Comment by bolololololo** [comment]
> You should see the obsidian icon folder settings when you go to settings. This is the same thing.

Exactly but it isn't there. I posted a screenshot of the plugin page because usually the "Options" button appear right next to "Disable" when the plugin has options but apparently it's a bug.

Anyway, it doesn't show up in the settings list too, so I can't get it to work:

![image](https://user-images.githubusercontent.com/80776140/235650029-8e6166c1-4934-4aaf-8343-51c74f24b144.png)

**Comment by bolololololo** [comment]
I closed the issue because I thought it was solved. I restarted Obsidian and the plugin was working, I set up a few icons, closed Obsidian again and when I reopened it the plugin wasn't working anymore.

It is enabled but it doesn't show the custom icons anymore and there is NOT a settings page.
![asdsad](https://user-images.githubusercontent.com/80776140/235701164-7f82e5eb-756e-46a9-8490-f76a12117868.png)
